libraries/standard/mqtt/cbmc/proofs/Makefile-project-defines

```diff
@@ -0,0 +1,34 @@
+# -*- mode: makefile -*-
+# The first line sets the emacs major mode to Makefile
+
+################################################################
+# Use this file to give project-specific definitions of the command
+# line arguments to pass to CBMC tools like goto-cc to build the goto
+# binaries and cbmc to do the property and coverage checking.
+#
+# Use this file to override most default definitions of variables in
+# Makefile.common.
+################################################################
+
+# Flags to pass to goto-cc for compilation (typically those passed to gcc -c)
+COMPILE_FLAGS += \
+        -fPIC \
+        -std=gnu90
+
+# Flags to pass to goto-cc for linking (typically those passed to gcc)
+LINK_FLAGS =
+
+# Preprocessor include paths -I...
+INCLUDES += \
+        -I$(SRCDIR)/libraries/standard/mqtt/cbmc/include \
+        -I$(SRCDIR)/libraries/standard/mqtt/include \
+        -I$(SRCDIR)/libraries/standard/utilities/include \
+        -I$(SRCDIR)/libraries/standard/mqtt/src \
+        -I$(SRCDIR)/demos/logging-stack \
+        -I$(SRCDIR)/demos/mqtt/mqtt_demo_basic_tls \
+        -I$(SRCDIR)/demos/mqtt/mqtt_demo_lightweight \
+        -I$(SRCDIR)/demos/mqtt/mqtt_demo_plaintext
```

Comment:
I think we should be able to include the `mqtt_demo_plaintext` demo only or have a separate config header file for these proofs.

Reply:
yes